morph/src/Response.re

```diff
@@ -48,82 +33,70 @@ let result_map = (fn, res) => {
   };
 };
 
-let empty = Ok({status: `OK, headers: [], body: String("")});
-
-let make = (~status=`OK, ~headers=[], body) => Ok({status, headers, body});
```

Comment:
has make been removed on purpose? Maybe it's a nice utility to have 🤔. It's still in rei. I must be missing something 😬

Reply:
I might have been tired when I did this, `empty` should be wrapped in a `Ok` and make should be `Piaf.Response.create` wrapped in a `Ok`, thank you for catching this.